README.md

```diff
@@ -1 +1,53 @@
-##Network Service
+# Network Service 
+
+The repo contains a Swift Package with the implementation of the HTTP network service for the iOS platform.
+
+⚠️ The Network service is under active development.
+
+## Table of contents
+
+* [General info](#general-info)
+* [Installation](#installation)
+* [Authors](#authors)
+
+## Current dev environment 
+
+| Dev environment| Ref link |
+| --- | -- |
+| ![macOS](https://img.shields.io/badge/macOS-13.3+-blue) |  
+| ![Xcode](https://img.shields.io/badge/Xcode-14.3-red)   |
+| ![Swift](https://img.shields.io/badge/Swift-5.8-orange) | 
+| ![iOS](https://img.shields.io/badge/iOS-15%2B-lightgrey) | 
+| ![SwiftFormat](https://img.shields.io/badge/SwiftFormat-0.51.7-yellow) | [Link](https://github.com/nicklockwood/SwiftFormat) |
+| ![SwiftLint](https://img.shields.io/badge/SwiftLint-0.51.0-green)| [Link](https://github.com/realm/SwiftLint) |
+
+## General info
+
+The repo contains the source code of the NetworkService.
+
+## Installation
+
+**Pre-requisites**: _installed Xcode with the version specified under the [current dev environment](#current-dev-environment)._
+
+To install Network Service package via Xcode
+
+1. Go to File -> Add Packages...
+1. Then search for https://github.com/ios-course/simple-network-service.git
+1. Click Add Package
+1. Leave default settings
```

Comment:
Could you apply a markdown format to improve readability?

Reply:
yes